### PR TITLE
Apply button fix for catalog item

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: -v -k 'test_delete_dialog_before_parent_item'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b439339f834c555311b2a982513787c20e2afabb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/services/catalogs/catalog_item.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
         ('edit_button', form_buttons.save),
         ('apply_btn', {
             version.LOWEST: '//a[@title="Apply"]',
-            version.LATEST: '//a[@data-original-title="Apply"]'})
+            '5.5': '//a[contains(text(),"Apply")]'})
```
Comment (psav) [inline cfme/services/catalogs/catalog_item.py]: Probably "normalize-space" is better here, could you replace and test?

Comment (akarol) [inline cfme/services/catalogs/catalog_item.py]: @psav replaced & tested.

**Comment by dajoRH** [comment]
Lint report for commit 74c67019579698b4f6b7d0b4d54d7b604a3e3a07:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**